SimplenoteUITests/Preview.swift

```diff
@@ -1,8 +1,10 @@
+import UITestHelpers
```

Comment:
This `import` is to use the functionality enabled by the `XCUIElementQuery` extension that's been moved into the framework.